.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What does this `.vs` folder mean? Do we really need this or it can be removed?

Reply:
Thank you, I've deleted this